data/categories.txt

```diff
@@ -4566,6 +4566,44 @@ vi:Phòng thí nghiệm y tế
 zh-Hans:医学实验室
 zh-Hant:醫學實驗室
 
+healthcare-physiotherapist
+en:Physiotherapist
```

Comment:
For the future reference: case in categories.txt file is not required, because these strings are used for search only, and are not visible to users.